updater/main.go

```diff
@@ -144,11 +148,14 @@ func _main() error {
 		activateErr := u.activateInstance(i.containerInstanceID)
 		if updateErr != nil && activateErr != nil {
 			log.Printf("Failed to update instance %#q: %v", i, updateErr)
+			summary[i.instanceID] = fmt.Sprintf("Failed to update and failed reactivate. Update error: %v, Aactivation error: %v", updateErr, activateErr)
```

Comment:
Updater will immediately stop after the return, therefore summary will not be printed and there is no point in additng it to summary.
```suggestion
			
```

Reply:
That's right, somehow that slipped my mind. Thanks for the call out!